Modules/ContentKit/Sources/Content/Categories/Category+Curation.swift

```diff
@@ -79,27 +79,37 @@ public extension CategoryCuration {
 
         public init(from decoder: any Decoder) throws {
             let container = try decoder.container(keyedBy: CodingKeys.self)
-            self.widgetType = try container.decode(WidgetType.self, forKey: .type)
+            self.layoutType = try container.decode(LayoutType.self, forKey: .layout)
+            self.widgetType = try container.decode(WidgetType.self, forKey: .widget)
             self.l10n = try container.decode([Category.LocalizedDetails].self, forKey: .l10n)
             self.items = try container.decode([Category.CurationPayload].self, forKey: .items)
         }
 
         // MARK: Public
 
         public enum CodingKeys: CodingKey {
-            case type
+            case layout
+            case widget
             case l10n
             case items
         }
 
+        public enum LayoutType: String, Codable {
+            case scrollgrid
+            case scroll
+            case grid
```

Comment:
what does those cases mean? what's the difference between them?

if possible, find inspiration from SwiftUI with HStack, VStack, etc.

Reply:
i would also add associated values like `scroll_grid`